backend/src/controller/defenceController.ts

```diff
@@ -254,14 +254,16 @@ function handleResetDefenceConfigItem(
 		currentDefences
 	);
 
-	const updatedDefences = req.session.levelState[level].defences;
-	if (updatedDefences === undefined) {
+	const updatedDefence = req.session.levelState[level].defences.find(
+		(defence) => defence.id === defenceId
+	);
+	if (!updatedDefence) {
```

Comment:
Change here is a result of eslint pointing out that `updatedDefences` in original code could not be undefined.